releasenotes/notes/default-schedule-name-51ba198cf08978cd.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixes a case in Result.get_counts(), where the results for an expirement
```

Also not a blocker but could you add a link to the `get_counts()` method:

```suggestion
    Fixes a case in :meth:`qiskit.result.Result.get_counts`, where the results for an expirement
```